compiler/parser/expression_parsing.go

```diff
@@ -388,6 +388,18 @@ func (p *Parser) parseMultiVariables(left ast.Expression) ast.Expression {
 	return result
 }
 
+func (p *Parser) parseDotExpression(receiver ast.Expression) ast.Expression {
+	_, ok := receiver.(*ast.IntegerLiteral)
+
+	// When both receiver & caller are integer => Float
+	if ok && p.peekTokenIs(token.Int) {
```

Comment:
Based on this code, expresions with spaces on the sides of the dot will be parsed as floats (eg. `2  .  3`).

If this is intended, I think it should be documented with a test case.

Reply:
Actually currently Goby doesn't recognize white space, so it's not just float has this issue, like:

```ruby
[] . length #=> 0
```

But this shouldn't be too hard to solve I think, already have a rough idea. But that won't be included in this PR.